naturalspeech2_pytorch/aligner.py

```diff
@@ -0,0 +1,143 @@
+from typing import Tuple
+import torch
+from torch import nn
+import torch.nn.functional as F
+from einops import rearrange
+import numpy as np
+class AlignerNet(torch.nn.Module):
+    """alignment model https://arxiv.org/pdf/2108.10447.pdf """
+    def __init__(
+        self,
+        dim_in=80,
+        dim_hidden=512,
+        attn_channels=80,
+        temperature=0.0005,
+    ):
+        super().__init__()
+        self.temperature = temperature
+        self.softmax = torch.nn.Softmax(dim=3)
+        self.log_softmax = torch.nn.LogSoftmax(dim=3)
```

Comment:
hmm, is this `log_softmax` being used anywhere?

Reply:
not used will remove this